cms/forms/wizards.py

```diff
@@ -78,10 +79,15 @@ class Media:
 
 
 class BaseCMSPageForm(forms.Form):
-    title = forms.CharField(label=_(u'Title'), max_length=255,
-                            help_text=_(u"Provide a title for the new page."))
-    page_type = forms.ChoiceField(label=_(u'Page type'), required=False,
-                                  widget=PageTypeSelect())
+    title = forms.CharField(
+        label=_(u'Title'), max_length=255,
+        help_text=_(u"Provide a title for the new page."))
+    slug = forms.CharField(
```

Comment:
Would it be possible to have such fields in an "Advanced options" fieldset? Slugs are definitely non-obvious to 99% of new users.